static/app/views/settings/projectPerformance/projectPerformance.tsx

```diff
@@ -149,6 +157,45 @@ class ProjectPerformance extends AsyncView<Props, State> {
     ];
   }
 
+  get performanceIssueDetectorsFormFields(): Field[] {
```

Comment:
Does any of this stuff need to be registered into the form sources? That way it's globally searchable

See https://github.com/getsentry/sentry/blob/dfe1d3442af1535cc2d4f8a511ee5733b3887572/static/app/actionCreators/formSearch.tsx#L42-L74

Reply:
Good question, but actually no, most of these will be just for us, since we need to tune thresholds etc. quite a bit, I'm going to add a feature flag to only make this available for devs after this.